test/lib/oauth.js

```diff
@@ -382,7 +397,8 @@ var setupAppConfig = function(config, callback) {
 
     var dummy = {
         close: function() {
-            child.kill();
+            child.kill("SIGKILL"); // SIGKILL really needed??
```

Comment:
Yeah, I can't really think of a reason that you should need this. So if you can test it without `SIGKILL` that'd be great, although if it still doesn't work somehow I'm fine with merging this and filing a followup